Assets/XML/Text/World Wonders.xml

```diff
@@ -2401,6 +2953,30 @@
+		<French>The [COLOR_BUILDING_TEXT]Chateau Frontenac[COLOR_REVERT] creates a free [COLOR_BUILDING_TEXT]Railway Station[COLOR_REVERT] in every city and increases the building speed for railroads. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Artist[COLOR_REVERT].</French>
+		<German>The [COLOR_BUILDING_TEXT]Chateau Frontenac[COLOR_REVERT] creates a free [COLOR_BUILDING_TEXT]Railway Station[COLOR_REVERT] in every city and increases the building speed for railroads. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Artist[COLOR_REVERT].</German>
+		<Italian>The [COLOR_BUILDING_TEXT]Chateau Frontenac[COLOR_REVERT] creates a free [COLOR_BUILDING_TEXT]Railway Station[COLOR_REVERT] in every city and increases the building speed for railroads. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Artist[COLOR_REVERT].</Italian>
+		<Spanish>The [COLOR_BUILDING_TEXT]Chateau Frontenac[COLOR_REVERT] creates a free [COLOR_BUILDING_TEXT]Railway Station[COLOR_REVERT] in every city and increases the building speed for railroads. It also increases the chance of generating a [COLOR_UNIT_TEXT]Great Artist[COLOR_REVERT].</Spanish>
+	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_OXFORD_UNIVERSITY_STRATEGY</Tag>
+		<English>[COLOR_BUILDING_TEXT]Oxford University[COLOR_REVERT] adds additional science to all of your specialists. It also increases its city's odds of generating a [COLOR_UNIT_TEXT]Great Scientist[COLOR_REVERT].</English>
+		<French>[COLOR_BUILDING_TEXT]Oxford University[COLOR_REVERT] adds additional science to all of your specialists. It also increases its city's odds of generating a [COLOR_UNIT_TEXT]Great Scientist[COLOR_REVERT].</French>
+		<German>[COLOR_BUILDING_TEXT]Oxford University[COLOR_REVERT] adds additional science to all of your specialists. It also increases its city's odds of generating a [COLOR_UNIT_TEXT]Great Scientist[COLOR_REVERT].</German>
+		<Italian>[COLOR_BUILDING_TEXT]Oxford University[COLOR_REVERT] adds additional science to all of your specialists. It also increases its city's odds of generating a [COLOR_UNIT_TEXT]Great Scientist[COLOR_REVERT].</Italian>
+		<Spanish>[COLOR_BUILDING_TEXT]Oxford University[COLOR_REVERT] adds additional science to all of your specialists. It also increases its city's odds of generating a [COLOR_UNIT_TEXT]Great Scientist[COLOR_REVERT].</Spanish>
+	</TEXT>
+	<TEXT>
+		<Tag>TXT_KEY_BUILDING_CRISTO_REDENTOR_STRATEGY</Tag>
+		<English>The [COLOR_BUILDING_TEXT]Cristo Redentor[COLOR_REVERT] allows its builder to switch [COLOR_HIGHLIGHT_TEXT]Religions[COLOR_REVERT] or [COLOR_HIGHLIGHT_TEXT]Civics[COLOR_REVERT] instantly once per turn, with no anarchy. It also increases the chances of a [COLOR_UNIT_TEXT]Great Prophet[COLOR_REVERT] appearing.</English>
```

Comment:
No article

Reply:
This is the original text from BTS. I only changed the Great Person type.